apps/web/src/app/(dashboard)/documents/data-table-action-button.tsx

```diff
@@ -37,38 +38,48 @@ export const DataTableActionButton = ({ row }: DataTableActionButtonProps) => {
   const isSigned = recipient?.signingStatus === SigningStatus.SIGNED;
 
   const onDownloadClick = async () => {
-    let document: DocumentWithData | null = null;
+    try {
+      let document: DocumentWithData | null = null;
 
-    if (!recipient) {
-      document = await trpcClient.document.getDocumentById.query({
-        id: row.id,
-      });
-    } else {
-      document = await trpcClient.document.getDocumentByToken.query({
-        token: recipient.token,
-      });
-    }
+      if (!recipient) {
+        document = await trpcClient.document.getDocumentById.query({
+          id: row.id,
+        });
+      } else {
+        document = await trpcClient.document.getDocumentByToken.query({
+          token: recipient.token,
+        });
+      }
 
-    const documentData = document?.documentData;
+      const documentData = document?.documentData;
 
-    if (!documentData) {
-      return;
-    }
+      console.log(documentData);
 
-    const documentBytes = await getFile(documentData);
+      if (!documentData) {
+        return;
+      }
 
-    const blob = new Blob([documentBytes], {
-      type: 'application/pdf',
-    });
+      const documentBytes = await getFile({ data: documentData.data, type: documentData.type });
 
-    const link = window.document.createElement('a');
+      const blob = new Blob([documentBytes], {
+        type: 'application/pdf',
+      });
+
+      const link = window.document.createElement('a');
 
-    link.href = window.URL.createObjectURL(blob);
-    link.download = row.title || 'document.pdf';
+      link.href = window.URL.createObjectURL(blob);
+      link.download = row.title || 'document.pdf';
```

Comment:
Is this missing the `_signed` change or am I missing something

Reply:
it was this pr #656, which has already been merged. this is just showing a toast on when there is an error to avoid breaking the application.